src/ServicePulse.Host/ServicePulse.Host.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net40</TargetFramework>
+    <TargetFramework>net45</TargetFramework>
```

Comment:
What a pity we couldn't change this to `netcoreapp3.0` as part of this taskforce, but I'm ok with it for now.